# Apparmor Video

rubencarneiro opened the pull request:
-  Allow Video Codecs and Video Playing

rubencarneiro committed:
# Apparmor Video
-  Allow Video Codecs and Video Playing

peat-psuwit commented:
The `/system/etc/media_codecs{,_*}.xml` should already be covered by [the main AppArmor template](https://gitlab.com/ubports/core/apparmor-easyprof-ubuntu/-/blob/main/data/templates/ubuntu/1.3/ubuntu-sdk#L479). For `media_profiles_V1_0.xml`, do you happen to know if this is something common or a device-specific thing? If it's common it might be better to bring this to the main template as well.

rubencarneiro commented:
Yes i found that makes videoplayer works on hammerhead, i applyied that to
the halium 7.1 to fix media player, also use that on halium 10 on redmi
note 8 pro to enable video playback.

Ratchanan Srirattanamet ***@***.***> escreveu em qua.,
20/04/2022 às 09:18 :

> The /system/etc/media_codecs{,_*}.xml should already be covered by the
> main AppArmor template
> <https://gitlab.com/ubports/core/apparmor-easyprof-ubuntu/-/blob/main/data/templates/ubuntu/1.3/ubuntu-sdk#L479>.
> For media_profiles_V1_0.xml, do you happen to know if this is something
> common or a device-specific thing? If it's common it might be better to
> bring this to the main template as well.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/ubports/android_device_lge_hammerhead/pull/31#issuecomment-1103605395>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJUAGM6OQVX5PTUDL6QXLK3VF64VJANCNFSM5TGUVSDA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

weirdal3333 commented:
Why hasn't this been merged in yet? The bug it addresses is almost 4 years old now, and this patch does indeed fix it.

rubencarneiro committed:
# Make initial changes to port halium 9
- ship usb moded
- ship halium-yaml with device info
- change folder and make a proper folder with halium and dir paths correctly so that user do know what is overlaying what.
- add gbinder.conf API level 29
- move ril configuration to ril_subscription.d so it dont get overlayed by OTA
- correct device.mk to copy new contents to correct folders
- move config to repowerd and rename to config-hammerhead

GermanAizek commented:
@rubencarneiro, wow.
perfect job, I was looking sources and wanted to make image 20.04 and fix config, but I found gold. You've made such big contribution to porting Nexus 5, but why hasn't it been merged yet?

@weirdal3333, agree wit.
